pyfakefs/fake_path.py

```diff
@@ -501,7 +500,7 @@ def __init__(self, filesystem: "FakeFilesystem"):
             Args:
                 filesystem: FakeFilesystem used to provide file system information
             """
-            import nt
+            import nt  # type:ignore[import]
```

Comment:
`pytype` does not see that this is in a specific platform branch - maybe we have to tell it some other way...